R/misc.R

```diff
@@ -1,9 +1,15 @@
-filter_terms <- function(x, ...) {
-  UseMethod("filter_terms")
+#' Translates into accepted recipes data types
+#' @details This function should be used for integration with other types of
+#' data sources. It is expected that this method will be used by other packages
+#' to provide a translation of the data types.
+#' @param x Table object used to determine data types
+#' @export
+recipes_translate_types <- function(x) {
+  UseMethod("recipes_translate_types")
 }
 
 ## Buckets variables into discrete, mutally exclusive types
-get_types <- function(x) {
```

Comment:
Take a look at #993 (specifically https://github.com/tidymodels/recipes/pull/993#discussion_r887321531). We might be expanding/refactoring how we do this part.

Reply:
This is the correct take. We should (once the dusts settles) make sure the whole `get_types()` system works well